index.html

```diff
@@ -4479,6 +4479,50 @@ <h2>HTTP</h2>
         <section class="informative">
           <h2>Credential Type-Specific Processing</h2>
 
+          <p>
+RDF is a technology for modeling graphs of statements. Each statement is a
+single <i>subject&#x2192;property&#x2192;value</i> (also known as
+<i>entity&#x2192;attribute&#x2192;value</i>) relationship, which is referred to
+as a <a>claim</a> in this specification. JSON-LD is a technology that enables
```

Comment:
Maybe add references to RDF and JSON-LD?
```suggestion
RDF [[?RDF11-CONCEPTS]] is a technology for modeling graphs of statements. Each statement is a
single <i>subject&#x2192;property&#x2192;value</i> (also known as
<i>entity&#x2192;attribute&#x2192;value</i>) relationship, which is referred to
as a <a>claim</a> in this specification. JSON-LD [[?JSON-LD11]] is a technology that enables
```